SokolTextGame/Commands.cs

```diff
@@ -39,7 +41,24 @@ public void Execute(string[] words, World world)
             if (string.Join(" ", words) == "look") Console.Write("Look... What's next? You can look at your weapon\n");
             else if (string.Join(" ", words) == "look at") Console.Write("Look at... what?\n");
             else if (string.Join(" ", words) == "look at my weapon") Console.Write(world?.player?.Weapon.Description);
-            else Console.Write("I know the command \"look at my weapon\"\n");
+            else if (string.Join(" ", words) == "look at guard")
```

Comment:
Same as in `talk to`, do not hardcode `talk to guard` here. Collapse this `if` and the next nested if (object name == words[2]) into one statement using `&&`. Check `if first two words are "look at" and the rest of the array is objectName` instead.